doc/swss-schema.md

```diff
@@ -106,7 +106,7 @@ For example (reorder output)
     oper_status         = "down" / "up"        ; operating status
     mtu                 = 1*4DIGIT             ; MTU for the IP interface of the VLAN
 
-    key                 = VLAN_TABLE:vlanid:ifname ; physical port member of VLAN
+    key                 = VLAN_MEMBER_TABLE:vlanid:ifname ; physical port member of VLAN
```

Comment:
In the current orchagent code, it does not read this tagging mode. You need this function, I think you can do that in another separate CR.